bigtable/metricscaler/metricscaler.py

```diff
@@ -16,10 +16,14 @@
 programmatically scale a Google Cloud Bigtable cluster."""
 
 import argparse
+import os
 import time
 
 from google.cloud import bigtable
-from google.cloud import monitoring
+from google.cloud import monitoring_v3
+from google.cloud.monitoring_v3.query import Query
+
+PROJECT = os.environ['GCLOUD_PROJECT']
```

Comment:
Usually we use `GOOGLE_CLOUD_PROJECT` but I forget if we set that for the Python tests.